### PR TITLE
Show Replies For Inquries when expanding

### LINKED ISSUE #245: Get the Replies For Inquries
Get the list of reply for specific inquiry when expanding.

### PR DESCRIPTION
**Pull request by pasindulakshan**
image
image

### CODE REVIEW

**Review by uihalagedara — APPROVED** [review]
LGTM

**Review by ThusharaX — APPROVED** [review]

**Review by KaveenLaksitha — COMMENTED** [review]
LGTM

**Review by miyurugunarathna — APPROVED** [review]
LGTM

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]
Can we add a boarder line to seperate message amd reply

File: src/pages/inquiry/list/index.tsx [hunk 1]
```diff
@@ -150,6 +150,19 @@ const InquiryList: React.FC = () => {
 						<p>{row.message}</p>
 					</div>
 				</div>
+
+				<h5>{translation["table-row-information"]["common-information"].reply}</h5>
+				{row.replies && row.replies.length ? (
+					row.replies.map((reply: any, index: number) => (
```
Comment (senuravihanjayadeva) [inline src/pages/inquiry/list/index.tsx:156]: we can give the type as a string instead of any. right?

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

File: src/pages/inquiry/list/index.tsx (hunk 1 above)
Comment (pasindulakshan) [reply src/pages/inquiry/list/index.tsx]: Yes

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**